src/util-privs.c

```diff
@@ -153,6 +153,10 @@ int SCGetUserID(const char *user_name, const char *group_name, uint32_t *uid, ui
     uint32_t groupid = 0;
     struct passwd *pw;
 
+    if (user_name == NULL || strlen(user_name) == 0) {
+        FatalError("user name cannot be set to an empty value");
```

Comment:
Sorry for being late to review this, but can we have something more user-oriented?
Something like `no user name was provided` ? I think something like that is easier to understand, from a non-dev perspective...

Similar for the other message 🤔